### DISCUSSION TITLE
Dockerized Support

### DISCUSSION THREAD

**Discussion by GregHilston** [ImRidingNow/PavlovRconWebserver#52] [Ideas]
Hey there @devinSpitz , any chance this application could be dockerized?

**Comment by ImRidingNow** [comment]
That is definitely possible. I will check the next few days and then write back here :)

**Comment by GregHilston** [comment]
Awesome! I'm excited to hear from you! :)

**Comment by ImRidingNow** [comment]
Sry for the long waiting. I have to fix a little more stuff on the test branch xD

When i release the version 0.0.3 i will have two options: 

- add a docker compose file to the repository
- Add an image to the docker hub

Im not sure what i will do but one of the options should be available in version 0.0.3 :)

**Reply by GregHilston** [reply]
No apology needed at all, you're the one developing open source software haha

Both options sound lovely :)

**Reply by ImRidingNow** [reply]
The Docker compose file is coming soon :D Just like the release to 0.0.3!

**Reply by GregHilston** [reply]
This is awesome!

**Reply by ImRidingNow** [reply]
Its there now :D

**Comment by GregHilston** [comment]
TY!

**Reply by ImRidingNow** [reply]
Feedback and questions are nice to see :)
